cmatrix.c

```diff
@@ -606,6 +639,16 @@ if (console) {
                 case '9':
                     update = keypress - 48;
                     break;
+                case 'm':
+		                space += 10;
```

Comment:
The indentation seems off in some places like this line for example. Can you please fix this